src/core/Chip/Chip.baseStyles.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme, SuomifiThemeProp } from '../theme';
+import { element, font, focus } from '../theme/reset';
+import { disabledCursor } from '../../components/utils/css';
+
+const removableStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-chip--removable {
```

Comment:
Are these really only for removable chips?

Reply:
Yes. When the icon is shown, the max width for the content has to be lower for the overflow styles and positioning to work correctly.